draft-ietf-homenet-front-end-naming-delegation.md

```diff
@@ -553,8 +559,9 @@ Uploading and dynamically updating the zone file on the DM can be seen as zone p
 This is handled using the normal zone transfer mechanism involving AXFR/IXFR.
 
 Part of this zone update process involves the owner of the zone (the hidden primary, the HNA) sending a DNS Notify to the secondaries.
-In this situation the only destination that is known by the HNA is the DM's Control Channel, and so DNS updates are to sent over the Control Channel, secured by TLS.
-DNS Notifies are not critical: they always cause the DM to use the Synchronization channel to do an SOA Query to detect any updates, and if there are some, then to transfer the zone.
+In this situation the only destination that is known by the HNA is the DM's Control Channel, and so DNS notifies are sent over the Control Channel, secured by TLS.
+
+However, DNS Notifies are not critical: they just cause the DM to use the Synchronization channel to do an SOA Query to detect any updates, and if there are some, then to transfer the zone.
```

Comment:
"However, DNS Notifies are not critical" I was expecting the text that follows to say that other channels 53 may be used as a consequence that Notifies are not critical. I do not read we give this possibility and I am wondering why we do have the "however".  I am readin gin fact the paragraph as a note, that is an information.

Reply:
I don't think that a DNS Notify to port 53 of the DM would necessarily mean anything.  It might not lead to the Synchronization system.  We don't say that 53 has to be alive, only 853.

Reply:
I agree. My only note was that I found strange "However: that seems to indicate something will follows that contradict what has just been said. That said, you know english better than I, so if "However" is fine, just ignore my comment.